pkg/config/config.go

```diff
@@ -130,10 +138,23 @@ func (c *Config) GetRPCsMap() *map[string]RPC {
 		rpcs[rpc.ChainName] = rpc
 	}
 
-	return &rpcs
+	// Validate RPCs exist for each IBC path
+	for _, ibcPath := range ibcPaths {
+		// Check RPC for chain 1
+		if _, ok := rpcs[ibcPath.Chain1.ChainName]; !ok {
+			return &rpcs, fmt.Errorf("missing RPC config for chain: %s", ibcPath.Chain1.ChainName)
```

Comment:
why not to use `ErrMissingRPCConfigMsg` like here https://github.com/archway-network/relayer_exporter/pull/32/files#diff-a3d824da3c42420cd5cbb0a4a2c0e7b5bfddd819652788a0596d195dc6e31fa5R150 ?